lldb/test/API/functionalities/data-formatter/swift-unsafe/TestSwiftUnsafeTypeFormatters.py

```diff
@@ -0,0 +1,63 @@
+# TestSwiftTypeAliasFormatters.py
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
```

Comment:
I remember 2016 fondly. For new tests, we can probably drop the copyright given they'll go stale.